Mathlib/Data/Nat/Basic.lean

```diff
@@ -707,6 +707,26 @@ protected theorem div_left_inj {a b d : ℕ} (hda : d ∣ a) (hdb : d ∣ b) : a
   rw [← Nat.mul_div_cancel' hda, ← Nat.mul_div_cancel' hdb, h]
 #align nat.div_left_inj Nat.div_left_inj
 
+theorem div_mul_div_comm {l : ℕ} (hmn : n ∣ m) (hkl : l ∣ k) :
+    m / n * (k / l) = m * k / (n * l) := by
```

Comment:
I think this is the victim of mathport dropping parens; it's a lot easier to parse as
```suggestion
    (m / n) * (k / l) = (m * k) / (n * l) := by
```